docs/units/logarithmic_units.rst

```diff
@@ -219,3 +219,10 @@ supported as logarithmic units.  For instance::
           <http://adsabs.harvard.edu/abs/1995PASP..107.1065H>`_
 .. [OG83] Oke, J.B., & Gunn, J. E., 1983, `ApJ 266, 713
 	  <http://adsabs.harvard.edu/abs/1983ApJ...266..713O>`_
+
+Reference/API
+=============
+.. automodapi:: astropy.units.function.logarithmic
```

Comment:
Is there a reason why this is also not in `units/index.rst` like `astropy.units.photometric` below?

Reply:
Good point, now done.